Modules/_multiprocessing/multiprocessing.c

```diff
@@ -139,35 +139,12 @@ static PyMethodDef module_methods[] = {
  * Initialize
  */
 
-static struct PyModuleDef multiprocessing_module = {
-    PyModuleDef_HEAD_INIT,
-    "_multiprocessing",
-    NULL,
-    -1,
-    module_methods,
-    NULL,
-    NULL,
-    NULL,
-    NULL
-};
-
-
-PyMODINIT_FUNC
-PyInit__multiprocessing(void)
+static int
+multiprocessing_exec(PyObject *module)
 {
-    PyObject *module, *temp, *value = NULL;
-
-    /* Initialize module */
-    module = PyModule_Create(&multiprocessing_module);
-    if (!module)
-        return NULL;
-
 #if defined(MS_WINDOWS) ||                                              \
   (defined(HAVE_SEM_OPEN) && !defined(POSIX_SEMAPHORES_NOT_ENABLED))
-    /* Add _PyMp_SemLock type to module */
-    if (PyType_Ready(&_PyMp_SemLockType) < 0)
```

Comment:
Did you try your PR? Python does crash with your PR because you removed this code.

You should either revert this removed call, or move  PyDict_SetItemString(_PyMp_SemLockType.tp_dict, ...) **after** PyModule_AddType() which calls PyType_Ready() for you.

Reply:
@vstinner no I was going to try it again today.  It was late at night I pushed this change.  I did notice the CI was broken